sessions/recap-project-6-NEW/activity-planner/README.md

```diff
@@ -0,0 +1,31 @@
+# Activity Planner
+
+## Description
+
+Activity Planner is a user-friendly application designed for outdoor enthusiasts. It aims to simplify the process of rediscovering and reliving individual outdoor activities by providing essential information and easy access to your favourite outdoor spots.
+
+### Main Features
+
+- **Activities List:** Browse a well-organised collection of activities.
+- **Create Activity:** Add new activities, enriching the activity catalogue.
+- **Activity Details:** View detailed information about each activity.
+- **Delete Activity:** Remove outdated or unwanted activities.
+- **Update Activity:** Update and modify existing activities.
```

Comment:
we can link to the respective user story .md files here so its easier to navigate the files.